src/sprites.py

```diff
@@ -1,5 +1,62 @@
 """ custom sprites classes """
 import pygame
+from src.settings import TILE_SIZE
+
+
+class Player:
+    """ move tile by tile """
+    def __init__(self):
+        # TODO: replace with actual images
+        self.image = pygame.Surface(size=(TILE_SIZE, TILE_SIZE))
+        self.image.fill('#ff0000')
+
+        self.rect: pygame.Rect = self.image.get_rect()
+
+        # this is used to only move onec when the mouse is pressed
+        self.mouse_have_been_pressed: bool = False
+
+    def update(self) -> None:
+        """ move the player """
+        if not pygame.mouse.get_pressed()[0]:
+            self.mouse_have_been_pressed = False
+            return None
+        if self.mouse_have_been_pressed:
+            return None
+
+        self.mouse_have_been_pressed = True
+        mouse_pos = pygame.mouse.get_pos()
+
+        delta_x = abs(self.rect.centerx - mouse_pos[0])
```

Comment:
What do you mean with delta? I see it is used to get the position of x and the position of Y. For others it could be useful to add a comment to what it does.

Reply:
yeah i'm gonna add comments but delta is a difference, it's to find out if the mouse is farther away on x axis or on y axis. To know where to move the player.